### ISSUE TITLE
SF CLI output defaults to the pickle format

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/solidfire-cli#36]
As subject. How to repro:

```sh
$ sfcli -m 192.168.1.34 -u admin -p admin volume list --limit 1
```

Config: 
- solidfire-cli           1.7.0.154           
- solidfire-sdk-python    12.3.0.203  
- Python 3.8.10

**Comment by arjun960** [comment]
@scaleoutsean yes, SF CLI output defaults to pickle format during installation of the cli, Please refer to Page No. 8 of the CLI User Guide ( https://github.com/solidfire/solidfire-cli/blob/master/NetAppElementCLIToolsUserGuide1_7.pdf ). 

To display the output of any cli command in JSON format we can pass -j or --json parameter along with other parameters of sfcli command.

**Comment by scaleoutsean** [comment]
Okay, I didn't read the PDF but I've read the [README.md](https://github.com/solidfire/solidfire-cli#tree-format-default) where it says tree format is the default and further below it says the pickle format requires `-k`.

**Comment by arjun960** [comment]
Yeah, you are right!! missed it completely, actually we missed to update the README.md file in the last release. We will update it soon.

Please refer to the [User Guide](https://github.com/solidfire/solidfire-cli/blob/master/NetAppElementCLIToolsUserGuide1_7.pdf ) for guidance related to sfcli.

**Comment by scaleoutsean** [comment]
Yeah the tree used to be the default for a while. IIRC maybe online help isn't up to date (I'm away from my PC now), as I don't recall seeing the switch for the tree format in online help.
